apps/argus/src/command/register_provider.rs

```diff
@@ -45,27 +43,13 @@ pub async fn register_provider_from_config(
         Arc::new(SignablePythContract::from_config(chain_config, &private_key_string).await?);
     // Create a new random hash chain.
     let random = rand::random::<[u8; 32]>();
-    let secret = provider_config
-        .secret
-        .load()?
-        .ok_or(anyhow!("Please specify a provider secret in the config"))?;
 
-    let commitment_length = provider_config.chain_length;
-    tracing::info!("Generating hash chain");
-    let chain = PebbleHashChain::from_config(
-        &secret,
-        chain_id,
-        &private_key_string.parse::<LocalWallet>()?.address(),
-        &chain_config.contract_addr,
-        &random,
-        commitment_length,
-        provider_config.chain_sample_interval,
-    )?;
-    tracing::info!("Done generating hash chain");
+    // FIXME: delete this
+    let commitment_length = 1000;
```

Comment:
this stuff will all get deleted later so I think it's fine to mock them out for now.